platform/security/policy/security-policy-context/src/main/resources/org.codice.ddf.security.policy.context.impl.PolicyManager.config

```diff
@@ -0,0 +1,27 @@
+authenticationTypes=[ \
+  "/\=IDP|GUEST", \
+  "/solr\=SAML|PKI|BASIC", \
+  ]
+realms=[ \
+  "/\=karaf", \
+  ]
+requiredAttributes=[ \
+  "/\=", \
+  "/admin\={http://schemas.xmlsoap.org/ws/2005/05/identity/claims/role\=system-admin}", \
+  "/solr\={http://schemas.xmlsoap.org/ws/2005/05/identity/claims/role\=system-admin}", \
+  "/system\={http://schemas.xmlsoap.org/ws/2005/05/identity/claims/role\=system-admin}", \
+  "/security-config\={http://schemas.xmlsoap.org/ws/2005/05/identity/claims/role\=system-admin}", \
+  ]
+traversalDepth=I"20"
+whiteListContexts=[ \
+  "${org.codice.ddf.system.rootContext}/SecurityTokenService", \
+  "${org.codice.ddf.system.rootContext}/internal/metrics", \
+  "/proxy", \
+  "/login", \
+  "/logout", \
+  "/idp", \
+  "${org.codice.ddf.system.rootContext}/idp", \
+  "${org.codice.ddf.system.rootContext}/saml", \
+  "/error", \
+  "${org.codice.ddf.system.rootContext}/platform/config/ui", \
+  ]
```

Comment:
✏️ Usually we have an extra empty line at the end of these config files because that's how felix formats them.

Reply:
Fixed